### PR TITLE
Bugfix My Resources Content shifts

### PR DESCRIPTION
**Pull request by Mygal-Alex**
https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/73c74e86-18ea-41c1-8761-4dd38167790e

### CODE REVIEW

**Comment by mhevyk** [comment]
Please link an issue to this PR

File: src/pages/create-or-edit-lesson/CreateOrEditLesson.styles.ts [hunk 1]
```diff
@@ -8,46 +21,54 @@ export const styles = {
...
+
+  descriptionLabel: (value: string) => ({
+    shrink: false,
     style: {
-      fontSize: '16px',
-      padding: '0',
-      marginTop: 0,
-      maxHeight: isDirty ? 'unset' : '16px'
+      visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,
+      color: palette.primary[300],
```
Comment (mhevyk) [inline src/pages/create-or-edit-lesson/CreateOrEditLesson.styles.ts:54]: The following files are duplicated:
```jsx
shrink: false,
style: {
  visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,
  color: palette.primary[300],
```

Please consider reusing it

Comment (Mygal-Alex) [reply src/pages/create-or-edit-lesson/CreateOrEditLesson.styles.ts:54]: Yes, I agree with you. 
I added it to the function.
fixed 

- [ ] fixed

**Review by D1nviz — APPROVED** [review]

**Review by asya982 — APPROVED** [review]

**Review by GeorgeShvab — APPROVED** [review]

**Comment by OliaHrytsak** [comment]
Don't forget to link issue to the pull request

**Review by mhevyk — APPROVED** [review]

**Review by byavd — APPROVED** [review]

**Pull request merged**